frontend/src/tests/GrantApplicationAdmin.spec.ts

```diff
@@ -0,0 +1,165 @@
+import router, { routes } from '@/router';
+import { mount, VueWrapper } from '@vue/test-utils';
+import { it, describe, beforeEach, expect, afterEach, vi } from 'vitest';
+import { routeItems } from '@/router/routeItem';
+import { fixJsdomCssErr } from './common/fixJsdomCssErr';
+import GrantApplicationAdmin from '@/components/grantaccess/GrantApplicationAdmin.vue';
+import waitForExpect from 'wait-for-expect';
+import { populateBreadcrumb } from '@/store/BreadcrumbState';
+
+fixJsdomCssErr();
+const mockRoutePush = vi.fn();
+vi.mock('vue-router', async () => {
+    const actual: Object = await vi.importActual('vue-router');
+    return {
+        ...actual,
+        useRoute: () => {
+            return routes.filter(
+                (route) => route.name == routeItems.grantAppAdmin.name
+            )[0];
+        },
+        useRouter: () => {
+            return {
+                push: mockRoutePush,
```

Comment:
Hi Jota, sorry just noticed this, do we need this useRouter?

Reply:
maybe we don't need this and the mockRoutePush, cause I think we have another routerPushSpy beblow

Reply:
it's not needed anymore, forget to remove after change some code, thanks!